dashboard/app/lib/javascripts/dashboard/main.js

```diff
@@ -65,6 +66,24 @@ window.Dashboard = {
 		Marbles.history.navigate('login'+ redirectPath);
 	},
 
+	__isCertInstalled: function() {
+		var self = this, dispatchIfCertIsMissing = function(a, b) {
+			if (a + b !== 200 + 200) {
```

Comment:
bold statement, because we could get 200 + 404 or something else here which does not imply an ssl/network error.